pandas/tests/indexing/test_iloc.py

```diff
@@ -827,7 +827,16 @@ def test_identity_slice_returns_new_object(self, using_array_manager):
         assert sliced_df is not original_df
 
         # should be a shallow copy
-        original_df["a"] = [4, 4, 4]
+        original_df.loc[:, "a"] = [4, 4, 4]
+        if not using_array_manager:
+            assert (sliced_df["a"] == 4).all()
+        else:
+            # FIXME: what is the expected/desired behavior here?  test it!
+            pass
+
+        # GH#35417 but setting with setitem creates a new array, so
+        #  sliced_df is not changed
+        original_df["a"] = [5, 5, 5]
```

Comment:
This doesn't raise a SettingWithCopy warning? Above a similar sliced case seems to assert a warning: https://github.com/pandas-dev/pandas/pull/35417/files#diff-f3753d3792b1cdb7c319380f7d1f58636ad0f9bbce037eb6920a6f9830412e9cR559

Reply:
`original_df["a"] = [5, 5, 5]` here is setting a new array, not overwriting an existing one, whereas the .loc settings in the linked tests are writing _into_ existing arrays